cloudman/data/update_keycloak.sh

```diff
@@ -14,14 +14,28 @@ realm_protection=$(curl -k -s -H "Content-Type: application/json" -H "Authorizat
 
 if [ "$realm_protection" = "false" ]
 then
-      
-      # Add Brute Force Detection to Master realm
+
+      # Add Brute Force Detection to 'master' realm
       curl -k -X PUT -H "Content-Type: application/json" -H "Authorization: bearer $token" {{ include "cloudman.root_url" . }}/auth/admin/realms/master -d '{"bruteForceProtected": true, "failureFactor": 5, "maxFailureWaitSeconds": 1800, "minimumQuickLoginWaitSeconds": 300}'
 
 else
       echo "Brute Force Protection is already on."
 fi
 
+# Get current user registration status
```

Comment:
I had typed this yesterday but forgot to "submit the review", so it was sitting in pending!

Reply:
What if we make the users not have any access by default, but they create their own accounts, and then admin has to give them access to cloudman/galaxy explicitly? When we have project-level access control this would work nicely too imo